SDK/AppCenterData/Microsoft.AppCenter.Data/PaginatedDocuments.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AppCenter.Data
+{
+    public partial class PaginatedDocuments<T>
+    {
+        /// <summary>
+        /// The current page.
+        /// </summary>
+        public Page<T> CurrentPage { get => null; }
+        
+        /// <summary>
+        /// Boolean indicating if an extra page is available.
+        /// </summary>
+        public bool HasNextPage { get => false; }
```

Comment:
Can you help me understand, if `HasNextPage` always returns false, is there any code setting it to true? How does it work?

Reply:
We don't provide the implementation for portable lib. It's just placeholder